proxysql_node_monitor

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 # This script will assist to setup Percona XtraDB cluster ProxySQL monitoring script.
 #####################################################################################
-
 HOSTGROUP_ID="${1:-0}"
 ERR_FILE="${2:-/dev/null}"
+SCHEDULER_ID="${3:-0}"
```

Comment:
With `proxysql-admin --config-file=/etc/proxysql-admin.cnf --enable` command SCHEDULER_ID will be always ZER0 and there wont any changes in scheduler table with  UPDATE statement.